src/content/whats-new/2022/02/InfraV2-WhatsNew.md

```diff
@@ -0,0 +1,27 @@
+---
+title: 'New Infrastructure Monitoring Experience is now Generally Available'
+summary: 'New time-travel and troubleshooting workflows for DevOps, SRE, and IT Ops teams make it possible to quickly pinpoint, isolate, and compare offending on-premises, cloud, or hybrid infrastructure components, determine incident blast radius, and identify root cause.'
+releaseDate: '2022-02-16'
+learnMoreLink: 'https://newrelic.com/blog/nerdlog/infrastructure-monitoring-in-preview'
+getStartedLink: 'http://one.newrelic.com/'
+---
+
+For Body:
+## New Infrastructure Monitoring is Aenerally Available Today
```

Comment:
Change title so that it's sentence case - New infrastructure monitoring is generally available today